### ISSUE TITLE
Tag names are not checked to be valid Java class names

### ISSUE THREAD

**Issue by gintautassulskus** [swagger-api/swagger-codegen#6482]
##### Description

I am trying to generate a Java client from the specification that has a tag, made up of digits and underscores. The codegen plugin generates an invalid Java class name _10Api.java_.

Is this the intended behaviour? If yes, which part of the swagger codegen would you advise extending in order to prefix the tag with characters so that the resulting Java class name would be valid, e.g. "Group 10Api"?

##### Swagger-codegen version

2.2.3

##### Swagger declaration file content or url

```
{
  "swagger": "2.0",
  "paths": {
    "/public": {
      "get": {
        "tags": [
          "1_0"
        ]
      }
    }
  }
}
```

##### Command line used for generation
Maven configuration is identical to that in here:
[https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen-maven-plugin/examples/java-client.xml](https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen-maven-plugin/examples/java-client.xml)

##### Suggest a fix/enhancement

Prefix tags that start with a digit with a generic keyword, e.g. "Tag/Group".

**Comment by wing328** [comment]
@gintautassulskus thanks for reporting the issue what about prefixing with "Class" (e.g. Class10) ?

**Comment by gintautassulskus** [comment]
@wing328 Yes, "Class" would be the most generic.

**Comment by wing328** [comment]
@gintautassulskus fixed via #6561. Please pull the latest master to give it a try.

**Comment by gintautassulskus** [comment]
@wing328, it works now. Many thanks.
